Rename beta SDKs to public preview SDKs in readme (#1998)
stripe/stripe-java: README.md

Stripe.enableTelemetry = false;
```

### Public Preview SDKs

Stripe has features in the [public preview phase](https://docs.stripe.com/release-phases) that can be accessed via the beta version of this package.
We would love for you to try these as we incrementally release new features and improve them based on your feedback.

> The public preview SDKs are just a different version of the same package and are appended with `-beta.X` such as `45.0.0-beta.1`. To install, choose the version that includes support for the preview feature you are interested in by reviewing the [releases page](https://github.com/stripe/stripe-java/releases/) and then use it [installation steps above](#installation).

Author: ```suggestion
The public preview SDKs are different version of the same package as the stable SDKs. These versions are appended with `-beta.X` such as `25.1.0-beta.1`. To install, choose the version that includes support for the preview feature you are interested in by reviewing the [releases page](https://github.com/stripe/stripe-java/releases/) and then use it [installation steps above](#installation).
```
